src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -165,6 +165,11 @@ unset VSCODE_NONCE
 # Report continuation prompt
 builtin printf "\e]633;P;ContinuationPrompt=$(echo "$PS2" | sed 's/\x1b/\\\\x1b/g')\a"
 
+__vsc_report_prompt() {
+	__vsc_prompt="$(builtin printf "%s" "${__vsc_original_PS1@P}" | sed 's/\x1b/\\\\x1b/g')"
```

Comment:
I'm not sure what this `@P` does?

Reply:
The `@P` expands the bash prompt so `\u` would become the username, [source](https://stackoverflow.com/questions/3451993/how-to-expand-ps1)

Reply:
Awesome🙂 could you add a comment explaining that in code?

Reply:
Added!